src/components/common/FloatEditor.vue

```diff
@@ -51,6 +52,17 @@ export default {
       required: false,
       default: '[0-9\-.,]',
     },
+    validationState: {
```

Comment:
I'm realising we repeat these part of code in other component. Wondering if you're happy to move it in a `mixins` way ? Like we did with `src/lib/filters/lang.ts`, we just need to do something similar in each component
```
// if class-based component, eg: components/interaction-designer/Block.vue
import { lang } from '@/lib/filters/lang'

export default {
  components: {
     ...
  }
  mixins: [lang],
  ...
}

// if class-based component, eg: components/interaction-designer/toolbar/TreeBuilderToolbar.vue
import Lang from '@/lib/filters/lang'

@Component({
  components: {
    ...
  },
})
export default class TreeBuilderToolbar extends mixins(Routes, Permissions, Lang) {
  ...
}
```
I can pair with you if you'd like we do this together.

Reply:
I am not sure if we still need this change as the code repetition has reduced significantly. Take a look at the new changes and let me know your thoughts.

Reply:
Let's skip for now @poojahpatil90 , we have less repetition than before after you made the 2nd changes. Thanks